sql/_34_fig/cbrd_24563/cases/cbrd_24563.sql

```diff
@@ -0,0 +1,28 @@
+-- CBRD-24563 : system parameter regexp_engine is added.
+-- Can choose the regexp library as 're2' or 'cppstd'.
+-- Verifies system parmeter in same session.
+
+-- positive case
+-- verify value : re2
+set system parameters 'regexp_engine=re2';
+select ('aabbb' rlike 'ab{0,49995}$');
+select ('aabbb' rlike 'ab{-1,3}$');
+
+-- verify value : cppstd
+set system parameters 'regexp_engine=cppstd';
+select ('aabbb' rlike 'ab{0,49995}$');
+select ('aabbb' rlike 'ab{-1,3}$');
+
+-- verify value : default (re2)
+set system parameters 'regexp_engine=default';
+select ('aabbb' rlike 'ab{0,49995}$');
+select ('aabbb' rlike 'ab{-1,3}$');
+
+-- negative case
+set system parameters 'regexp_engine=' '';
```

Comment:
@hgryoo 
if the value is empty should occur an error.
But now it works the same as the former set value.

Reply:
The regexp_engine is a PRM_KEYWORD type parameter. If an invalid value is specified, CUBRID ignores it.
Unfortunately, there seems to be no description of the keyword type parameters... :-(

The following parameters are also keyword types like regexp_engine:
```
error_log_level
isolation_level
page_validation_level
ha_mode_for_sa_utils_only
ha_server_state
ha_log_applier_state
compat_mode
check_peer_alive
query_trace_format
fault_injection_test
ha_repl_filter_type
tde_default_algorithm
```

Reply:
The description is added in the test case.